src/Store/auth-context.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+
+const AuthContext = React.createContext({
+    token: '',
+    isLoggedIn: false,
+    login: (token) => { },
+    logout: () => [],
+    isAdmin: false,
+});
+
+export const AuthContextProvider = (props) => {
+    const initialToken = localStorage.getItem('token');
+    const [token, setToken] = useState(initialToken);
+
+    const [admin,setAdmin] = useState(false);
+
+    const userIsLoggedIn = !!token;
+
+    const loginHandler = (props) => {
```

Comment:
Nice! Ik wil alleen even meegeven dat in de praktijk volgens mij vaak handleLogin wordt gebruikt. Dus eerst 'handle' en dan vervolgens wat er dan afgehandeld wordt.

Reply:
Ik denk dat dit dan gebruik is van verschillende programmeurs, ik heb voor het grootste gedeelte de logica van een React Udemy course genomen. En die gaf aan dat alles mag, maar dat dit goed gebruik was